tests/integration/zombie_nomnom_api/test_server.py

```diff
@@ -14,3 +14,11 @@ def test_cors_function_for_api(api_client: TestClient):
         "/healthz", headers={"origin": "http://localhost:3000"}
     )
     assert response.headers.get("access-control-allow-origin") == "*"
+
+
+def test_version__when_requesting_version__returns_object_with_version_field(
+    api_client: TestClient,
+):
+    response: httpx.Request = api_client.get("/version")
+    value = json.loads(response.content)
+    assert "version" in value
```

Comment:
Should check that the value at the version key is populated and not null

Reply:
fixed